### PR TITLE
trace - Add TimeToFirstByte to output in --json mode

### PR DESCRIPTION
**Pull request by poornas**
ttfb is now available in the trace API  call via PR https://github.com/minio/minio/pull/8283, so adding this to trace output in json mode.

### CODE REVIEW

File: cmd/admin-trace.go [hunk 1]
```diff
@@ -171,6 +171,7 @@ type callStats struct {
 	Rx       int           `json:"rx"`
 	Tx       int           `json:"tx"`
 	Duration time.Duration `json:"duration"`
+	Ttfb     time.Duration `json:"timetofirstbyte"`
```
Comment (krisis) [inline cmd/admin-trace.go]: It is usually helpful to have camel case or snake case for multi-word JSON keys. It simplifies consuming JSON values of this type in languages other than Go.

Comment (poornas) [reply cmd/admin-trace.go]: good suggestion - changed the rest of the multi word keys to camel case also.

File: go.mod [hunk 2]
```diff
@@ -4,24 +4,22 @@ go 1.13
...
 	github.com/fatih/color v1.7.0
 	github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
 	github.com/mattn/go-colorable v0.1.1
 	github.com/mattn/go-isatty v0.0.7
 	github.com/minio/cli v1.21.0
-	github.com/minio/minio v0.0.0-20190903181048-8a71b0ec5a72
-	github.com/minio/minio-go/v6 v6.0.36
-	github.com/minio/sha256-simd v0.1.0
+	github.com/minio/minio v0.0.0-20191001201215-ff5bf519522f
+	github.com/minio/minio-go/v6 v6.0.37
```
Comment (harshavardhana) [inline go.mod]: We can make a new release and then update this helpful for other bugs we saw in MinIO and mc @kannappanr ??

**Comment by kannappanr** [comment]
Please resolve the go.mod conflicts

**Pull request merged**